Smarty/templates/ActivityReminderCallback.tpl

```diff
@@ -15,7 +15,7 @@
 	<tr>
 		<td align='left'><b>
 		<img align="top" src="themes/images/{$activityimage}s.gif"/> {$activitytype} - {$cbstatus}</b> </td>
-		<td align='right'><b><font color={$cbcolor}>{$cbdate} {$cbtime}</font></b></td>
+		<td align='right'><b><div color={$cbcolor}>{$cbdate} {$cbtime}</div></b></td>
```

Comment:
this doesn't look correct, shouldn't it be

`<span style="color:{$cbcolor}">`

?